animations/transition_widgets.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: "Flutter transition animations"
+permalink: /animations/transitions_widgets.html
+---
+
+* TOC Placeholder
+{:toc}
+
+Transition animations implement the abstract class `AnimatedWidget`. The `AnimatedWidget` class allows you to separate the widget code from the animation code in the `setState()` call. The `AnimatedWidget` class doesn’t need to maintain a `State` object to hold the animation. Use the  `AnimatedWidget` helper class (instead of `addListener()` and `setState()`) to create a reusable transition animation widget.
+
+< additional info on transition widgets >
+
+
+## Using a transition animation widget
+
+<div>
+<table class="table" width="100%">
+  <col width="35%">
+  <col width="65%">
+	<tbody>
+    <tr>
+      <td><img src="images/transitions.png" alt="Transition animation widgets"></td>
+      <td> < Revise for a transition widget example >   
+      1. Create your <code>StatefulWidget</code>.<br>
+      2. Create the <code>State</code> class to hold the animation object and the _transition_  animation widget.<br>
+      3. Specify the <code>BuildContext</code> to return the _transition_ animation widget, such as <code>animationexample</code>.<br>
+      4. Add the animation object, the duration of the animation, and any parameters relating to the _transition_ animation. For example, you would set the < example > for a <code>animationexample</code> widget.
```

Comment:
As I mentioned in the implicit animation doc, steps 1-3 is just how you create any stateful widget.